Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/ThrowDaggersInBensEyes.java

```diff
@@ -44,14 +44,14 @@ public void initDefaultCommand() {
         //setDefaultCommand(new MySpecialCommand());
     }
 
-	public void reverse(Object object) {
+	//public void reverse(Object object) {
 		// TODO Auto-generated method stub
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)